### PR TITLE
Add research document for tools like AI Verify

### PR DESCRIPTION
**Pull request by uittenbroekrobbert**
# Description

@ChristopherSpelt and myself did a (re)search on TAD tools similar to AI Verify. This document is the result of that research, containing a shortlist of tools we may investigate further and also includes links to all kinds of tooling and information that can be useful.

### CODE REVIEW

**Review by berrydenhartog — DISMISSED** [review]
Can you combine the Commits nicely?

**Review by berrydenhartog — APPROVED** [review]
LGTM

File: docs/Projects/TAD/tools.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+tools to measure and mitigate bias across numerous tasks. Will be extended to include tools for
+efficacy, robustness, privacy and explainability.
+
+**Why interesting?** Although it is not entirely clear what exactly this tool does (see Remarks)
+it does seem (according to their website) to provide reports on bias and fairness. The Github rep
+does not seem to include any report generating code, but mainly technical tests.
+[Here](https://holisticai.readthedocs.io/en/latest/tutorials/measuring_bias_tutorials/measuring_bias_classification.html)
+is an example in which bias is measured in a classification model.
+
+**Remarks** Webiste seems to suggest to possibility to generate reports, but this is not directly
```
Comment (anneschuth) [inline docs/Projects/TAD/tools.md:61]:
```suggestion
**Remarks** Website seems to suggest to possibility to generate reports, but this is not directly
```

File: docs/Projects/TAD/tools.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## AI Verify
+
+AI Verify is an AI governance testing framework and software toolkit that validates the performance of AI systems against
+a set of  internationally recognised principles through standardised tests, and is consistent with international AI governance
+frameworks such as those from European Union, OECD and Singapore.
+
+Links:
+[AI Verify Homepage](https://aiverifyfoundation.sg/),
+[AIVerify documentation](https://imda-btg.github.io/aiverify/),
+[AIVerify Github](https://github.com/IMDA-BTG/aiverify).
```
Comment (anneschuth) [inline docs/Projects/TAD/tools.md:18]:
```suggestion
[AI Verify Documentation](https://imda-btg.github.io/aiverify/),
[AI Verify Github](https://github.com/IMDA-BTG/aiverify).
```

**Review by anneschuth — DISMISSED** [review]
Some small comments, LGTM!

File: docs/Projects/TAD/tools.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+**Remarks** Webiste seems to suggest to possibility to generate reports, but this is not directly
+reflected in the codebase. Possibly reports are only available with some sort of licenced product?
+
+Links:
+[Hollisticai homepage](https://www.holisticai.com/),
+[Hollisticai Github](https://github.com/holistic-ai/holisticai).
+
+## Interesting to mention
+
+* [What-if](https://github.com/pair-code/what-if-tool)*. Provides interface for expanding understanding
```
Comment (robbertbos) [inline docs/Projects/TAD/tools.md:70]:
```suggestion
* [What-if](https://github.com/pair-code/what-if-tool). Provides interface for expanding understanding
```

File: docs/Projects/TAD/tools.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Links: [ALTAI ai4belgium Homepage](https://altai.ai4belgium.be/),
+[Altai Github](https://github.com/AI4Belgium/ai-assessment-tool).
+
+### IBM Research 360 Toolkit
+
+**What is it?** Open source Python libraries that supports interpretability and explainability of
+datasets and machine learning models. Most relevant tookits are the AI Fairness 360 and AI Explainability 360.
+
+**Why interesting?** Seems to encompass extensive fairness and explainabiity tests. Codebase seems
```
Comment (robbertbos) [inline docs/Projects/TAD/tools.md:41]:
```suggestion
**Why interesting?** Seems to encompass extensive fairness and explainability tests. Codebase seems
```

File: docs/Projects/TAD/tools.md (hunk 1 above)
Comment (robbertbos) [reply docs/Projects/TAD/tools.md:61]:
```suggestion
**Remarks** Website seems to suggest the possibility to generate reports, but this is not directly
```

File: docs/Projects/TAD/tools.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+package which enables easy computation of commonly-identified fairness metrics for binary and
+multiclass classifiers. Part of TensorFlow.
+k
+* [Fairlearn](https://github.com/fairlearn/fairlearn). Open source Python package that empowers
+developers of AI systems to assess their system's fairness and mitigate any observed unfairness
+issuess.
+
+* [Dalex](https://dalex.drwhy.ai/). The DALEX package xrays any model and helps to explore and
+explain its behaviour, helps to understand how complex models are working. The main function
+explain() creates a wrapper around a predictive model. Wrapped models may then be explored and
```
Comment (robbertbos) [inline docs/Projects/TAD/tools.md:91]:
```suggestion
`explain()` creates a wrapper around a predictive model. Wrapped models may then be explored and
```

File: docs/Projects/TAD/tools.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+compared with a collection of local and global explainers. Recent developents from the area of
```
Comment (robbertbos) [inline docs/Projects/TAD/tools.md:92]:
```suggestion
`explain()` creates a wrapper around a predictive model. Wrapped models may then be explored and
compared with a collection of local and global explainers. Recent developments from the area of
```

**Comment by robbertbos** [comment]
@uittenbroekrobbert thanks for the summary of the research done. I left some suggestions, but in general LGTM!

File: docs/Projects/TAD/tools.md (hunk 1 above)
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

File: docs/Projects/TAD/tools.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+Links:
+[AI Verify Homepage](https://aiverifyfoundation.sg/),
+[AIVerify documentation](https://imda-btg.github.io/aiverify/),
+[AIVerify Github](https://github.com/IMDA-BTG/aiverify).
+
+## To investigate further
+
+### AI Assessment Tool Belgium
+
+**What is it?** The tool is based on the ALTAI recommendations published by the European Commission
```
[the remark this answers is not in the document]
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

File: docs/Projects/TAD/tools.md (hunk 6 above)
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

File: docs/Projects/TAD/tools.md (hunk 4 above)
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

File: docs/Projects/TAD/tools.md (hunk 2 above)
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

File: docs/Projects/TAD/tools.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+* [Anch.ai](https://anch.ai/about/). The end-to-end cloud solution empowers global data-driven
+organizations to govern and deploy responsible, transparent, and explainable AI aligned with
+upcoming EU regulation AI Act. Does not seem to be open source.
+
+* [CredoAI](https://www.credo.ai/). Credo AI is an AI governance platform that helps companies adopt,
+scale, and govern AI safely and effectively. Does not seem to be open source.
+
+## The FATE system
+
+Paper by TNO, acronym FATE stands for "FAir, Transparent and Explainable Decision Making."
```
[the remark this answers is not in the document]
Comment (ChristopherSpelt) [reply docs/Projects/TAD/tools.md]: Done

**Review by laurensWe — APPROVED** [review]
LGTM :)

**Review by anneschuth — APPROVED** [review]

**Pull request merged**